CommunityCall/notes/20220217_Notes.md

```diff
@@ -0,0 +1,60 @@
+# PowerShell Community Call - February 17, 2022
+
+## Agenda
+
+- new WG members (Michael)
+- docs update (Sean)
+- moving to .NET 7 (Aditya)
+- demo the completion-based predictor (Dongbo)
+- VSCode extension update (Andy)
+
+## Notes
+
+- Welcoming new [New WG Members](https://github.com/PowerShell/PowerShell/commit/7dd0c43fb391c322d2ed01f1df26154a765455f8)
+  - Benchmark working group, DSC WG, few others
+  - Few new committee level members

```

Comment:
Was docs updated?

Reply:
> Was docs updated?

Yeah are there actually any new committee members? Or was that comment based on a misleading diff?

![image](https://user-images.githubusercontent.com/24977523/154736059-4350f565-7b69-4422-895f-c786b60eda36.png)

The above is actually referencing the cmdlets work group. The PowerShell Committee is a line item under that group, not describing what the rest are. (note, I read it the same way. Very unfortunate diff 😁 )

Reply:
@theJasonHelmick There is no committee level changes, right? Can you please update this note to remove this line?

Reply:
Oops that might have been my mistake, I must have misheard when taking notes, I believe I meant to say Working group members

Reply:
updated